File: 240418/좀 더 어려운 수학 점수/math-scores-are-more-difficult.js
const fs = require('fs');
const [mathA, englishA, mathB, englishB] = fs.readFileSync(0).toString().split(/\s+/).map(Number);

if(mathA === mathB){
    if(englishA > englishB){
        console.log('A');
    }else{
        console.log('B');
    }
}else{
    if(mathA > mathB){
        console.log('A');
    }else{
        console.log('B');
    }
}

File: 240418/좀 더 어려운 수학 점수/math-scores-are-more-difficult.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int mathA = sc.nextInt();
        int englishA = sc.nextInt();

        int mathB = sc.nextInt();
        int englishB = sc.nextInt();

        if(mathA === mathB){
            if(englishA > englishB){
                System.out.println('A');
            }else{
                System.out.println('B');
            }
        }else{
            if(mathA > mathB){
                System.out.println('A');
            }else{
                System.out.println('B');
            }
        }
    }
}